### PR TITLE
Use cdnjs

### PR DESCRIPTION
**Pull request by sirahd**
-Use cdnjs link in all `.md` files under `/site`
-Write bash script to get current `vega` and `vega-lite` version and update `/_data/versions.yml`

### CODE REVIEW

**Comment by kanitw** [comment]
Not sure why u have to close and open new PR.

In general, you can use force push for ur own branch :)

**Comment by sirahd** [comment]
I tried renaming Dom's branch to my name, but it keeps resetting to the previous name. So that's why I delete his branch and create a new one. Is there a better way to do this?

**Comment by NeelMohapatra** [comment]
http://stackoverflow.com/questions/6591213/how-to-rename-a-local-git-branch
This might help

**Comment by kanitw** [comment]
I guess you mean you want the PR to use your branch, but since it was created from dom's branch, there is no way to make the old use a new branch.

I would say in general, you can continue to use his branch, but if you really want the branch to be your name then closing and reopen is the only way.

**Comment by sirahd** [comment]
@kanitw that's exactly what I'm trying to do! I thought using his branch might confuse other people, but if that's totally fine, I'll continue on the same branch next time!

**Comment by domoritz** [comment]
@sirahd It's fine to continue using a branch once we created it.

File: package.json [hunk 1]
```diff
@@ -25,13 +25,14 @@
...
     "build:toc": "bundle exec jekyll build --incremental -q && scripts/generate-toc",
+    "build:versions": "scripts/update-version.sh",
     "build:test-gallery": "browserify test-gallery/main.ts -p [tsify -p test-gallery] -d > test-gallery/main.js",
     "checkschema": "./scripts/check-schema.sh",
     "clean": "rm -f vega-lite.* & find -E src test site -regex '.*\\.(js|js.map|d.ts)' -delete & rm -rf examples/_diff examples/_original examples/_output examples/images && rm -rf data",
     "data": "rsync -r node_modules/vega-datasets/data/* data",
     "deploy": "scripts/deploy.sh",
     "deploy:gh": "scripts/deploy-gh.sh",
-    "deploy:schema": "scripts/deploy-schema.sh",
+    "deploy:schema": "./scripts/deploy-schema.sh",
```
Comment (domoritz) [inline package.json]: Why is this necessary?

File: scripts/update-version.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+FILE='../_data/versions.yml'
+
+echo "vega: `npm info vega version`" > $FILE
+echo "vega-lite: `npm info vega-lite version`" >> $FILE
+echo "vega-embed: `npm info vega-embed version`" >> $FILE
```
Comment (domoritz) [inline scripts/update-version.sh]: we usually have an empty line at the end of ech file

File: site/demo.html [hunk 3]
```diff
@@ -42,17 +42,11 @@
...
   // Embed the visualization in the container with id `vis`
-  vg.embed("#vis", embedSpec, function(error, result) {
+  vg.embed("#vis", vlSpec, function(error, result) {
     // Callback receiving the View instance and parsed Vega spec
     // result.view is the View, which resides under the '#vis' element
   });
   </script>
 </body>
-</html>
+</html>
```
Comment (domoritz) [inline site/demo.html]: Same as above. If you install the editorconfig extension, this should be done automatically.

File: site/tutorials/gettings_started.md [hunk 4]
```diff
@@ -249,10 +249,10 @@ You can use [Vega-Embed](https://github.com/vega/vega-embed) to embed your Vega-
   <title>Vega Lite Bar Chart</title>
   <meta charset="utf-8">
 
-  <script src="https://d3js.org/d3.v3.min.js"></script>
-  <script src="https://vega.github.io/vega/vega.js"></script>
-  <script src="https://vega.github.io/vega-lite/vega-lite.js"></script>
-  <script src="https://vega.github.io/vega-editor/vendor/vega-embed.js" charset="utf-8"></script>
+  <script src="//d3js.org/d3.v3.min.js"></script>
```
Comment (domoritz) [inline site/tutorials/gettings_started.md]: add `https:` prefix to all of these urls.

File: site/usage/embed.md [hunk 5]
```diff
@@ -13,9 +13,9 @@ To embed a Vega-Lite specification on your web page first load the required libr
 
 ```html
 <script src="//d3js.org/d3.v3.min.js" charset="utf-8"></script>
```
Comment (domoritz) [inline site/usage/embed.md]: same as above

File: package.json (hunk 1 above)
Comment (sirahd) [inline package.json]: This is typo ... I didn't mean to change it

File: site/demo.html (hunk 3 above)
Comment (sirahd) [inline site/demo.html]: Also, how do I change html file so that every time the script gets called, the url is also updated. Currently it only works in markdown file

Comment (domoritz) [inline site/demo.html]: You have to add the header for jekyll. I think

```
---
---
```

may be sufficient.

File: package.json (hunk 1 above)
Comment (domoritz) [inline package.json]: No problem. I realize I reviewed code before you said you were done but I thought it would be useful for you to get early feedback.

**Comment by sirahd** [comment]
@domoritz Done!

**Comment by domoritz** [comment]
🎉
